Lib/test/_test_multiprocessing.py

```diff
@@ -3880,6 +3880,10 @@ class OptionalAttachSharedMemory(shared_memory.SharedMemory):
 
         sms.close()
 
+        # Test creating a shared memory segment with size greater than shared_memory._MAX_SHM_SIZE_LIMIT
```

Comment:
I don't think there's a need for this comment.

Reply:
I added this comment because similar tests around this test use such comments.